lib/ts/controllers/s-popover.ts

```diff
@@ -194,6 +194,15 @@ namespace Stacks {
         private boundHideOnOutsideClick!: any;
         private boundHideOnEscapePress!: any;
 
+        connect() {
+            super.connect();
+
+            if (this.data.get("show-on-connect") === "true") {
+                this.data.delete("show-on-connect");
```

Comment:
Why delete the data after showing? You'd want it to re-run if the controller was ever reconnected.

Reply:
I could rename it to `show-on-first-connect` to reduce ambiguity.  This attribute is meant mainly for `show` and when you call `show`, you indend for it to fire once, not many times.

Reply:
That's fair enough. I was more worried about the `remove, then readd to dom` circumstance, but... that's not likely to happen often and even then, I'm ok with not auto showing